cv_next/middleware.ts

```diff
@@ -74,4 +74,4 @@ export async function middleware(request: NextRequest) {
   return response;
 }
 
-export const config = { matcher: [] };
+export const config = { matcher: ["/cv/:cvId*"] };;
```

Comment:
generate route with values, don't hardcode it; @Aradon267 similar to what I've suggested about api routes

Reply:
Updated here in a somewhat similar fashion as I did in the other branch(there I did everything in the hook while here I created another class in the definitions and also moved some things around
lmk if these versions are fine

Reply:
For everything I try and do instead of hardcoding it(putting it in definitions, as a const above, etc) generates:
```
Next.js can't recognize the exported `config` field in route "/middleware":
Unsupported node type "MemberExpression" at "config.matcher".
The default config will be used instead.
Read More - https://nextjs.org/docs/messages/invalid-page-config
```